tests/test_utils.py

```diff
@@ -1,36 +1,57 @@
 # -*- coding: utf8 -*-
+import unittest
 
 from pyoes.utils import (
     set_attr_filter,
     datetime_format_filter
-    )
+)
 
-class TestSetAttr:
 
+class TestSetAttr(unittest.TestCase):
     def test_set_attr_filter(self):
         d = {}
         d2 = set_attr_filter(d, 'pagina', 1)
-        assert d2 == {'pagina': 1}
+        self.assertEqual({'pagina': 1}, d2)
 
     def test_set_attr_filter_overwrites(self):
         d = {'pagina': 1}
         d2 = set_attr_filter(d, 'pagina', 10)
-        assert d2 == {'pagina': 10}
+        self.assertEqual({'pagina': 10}, d2)
 
-class TestDatetimeFormatFilter:
 
+class TestDatetimeFormatFilter(unittest.TestCase):
     def test_datetime_format_filter_default(self):
         from datetime import datetime
         d = datetime(2014, 9, 7, 20, 00)
         fd = datetime_format_filter(d)
-        assert fd == '07-09-2014 20:00'
+        self.assertEqual('07-09-2014 20:00', fd)
 
     def test_datetime_format_filter_custom_format(self):
```

Comment:
Kunnen we nog een gelijkaardige test toevoegen met een datetime.date ipv een datetime.datetime?